### DISCUSSION TITLE
Is it me or has 2023.6 broken the EM integration???

### DISCUSSION THREAD

**Discussion by smoketabs** [nathanmarlor/foxess_em#303] [General]
Hi.

Has anyone had problems with etc EM integration after updating to 2023.6?

Or is it just me?

Thanks.

**Comment by stevie445** [comment]
Yip.  Me 2

**Reply by smoketabs** [reply]
> Yip. Me 2

phew... Thought it was just me.

I'm liking all the new icons, mind you.

**Comment by spmaid** [comment]
Maybe not related, but I note on the website, the 'force charge' option has gone, only 'Enable charge from grid'.  In the App though, it's still there



 







---- On Fri, 09 Jun 2023 07:38:25 +0100 smoketabs ***@***.***> wrote ---





Yip. Me 2


phew... Thought it was just me.

I'm liking all the new icons, mind you.

**Comment by nathanmarlor** [comment]
Can you try -> https://github.com/nathanmarlor/foxess_em/releases/tag/v1.8.0 please?
